sap-s4hana-cloud/2.2/modules/ROOT/pages/index.adoc

```diff
@@ -0,0 +1,60 @@
+= SAP S/4HANA OData Connector 2.2 - Mule 4
+:page-aliases: connectors::sap/sap-s4hana-cloud-connector.adoc
+
+Support Category: https://www.mulesoft.com/legal/versioning-back-support-policy#anypoint-connectors[Premium]
+
+Anypoint Connector for SAP S/4HANA OData Cloud (SAP S/4HANA OData Cloud Connector) provides support for the OData APIs exposed by SAP S/4HANA Cloud and SAP S/4HANA on-premise.
```

Comment:
Please remove Cloud word from parenthesis where connector is mentioned and try to explicitly say that on-Premise support is available from version 2.1.x.